e2e_tests/conftest.py

```diff
@@ -1,17 +1,37 @@
 import logging
+import subprocess
+import uuid
+
+import docker
 import pytest
 from sklearn import svm, datasets
 
+from bentoml.deployment.aws_lambda import _cleanup_s3_bucket_if_exist
+from bentoml.deployment.utils import ensure_docker_available_or_raise
+from bentoml.utils.s3 import create_s3_bucket_if_not_exists
+from bentoml.utils.tempdir import TempDirectory
+from e2e_tests.iris_classifier_example import IrisClassifier
+from e2e_tests.cli_operations import delete_bento
 from e2e_tests.basic_bento_service_examples import (
     BasicBentoService,
     UpdatedBasicBentoService,
 )
-from e2e_tests.iris_classifier_example import IrisClassifier
-from e2e_tests.cli_operations import delete_bento
 
 logger = logging.getLogger('bentoml.test')
 
 
+def wait_until_container_is_running(container_name):
+    docker_client = docker.from_env()
+    container_is_not_running = True
+    while container_is_not_running:
```

Comment:
```
while True:
    ...
    for container in container_list:
        if container.name == container_name:
            break
```

Reply:
two loops can't use break just for the inner one